# SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
# Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
# SPDX-License-Identifier: Apache-2.0

mega_node_monitor_service:
  ros__parameters:
    monitored_nodes:
      - navigation_container
      - nova_container

    monitored_lifecycle_nodes:
      - behavior_server
      - bt_navigator
      - controller_server
      - global_costmap/global_costmap
      - local_costmap/local_costmap
      - planner_server
      - smoother_server
      - velocity_smoother
      - waypoint_follower
